clients/src/main/java/org/apache/kafka/common/config/ConfigDef.java

```diff
@@ -1070,8 +1081,10 @@ private void validateIndividualValues(String name, List<Object> values) {
         }
 
         public String toString() {
-            return validString + (isEmptyAllowed ? " (empty config allowed)" : " (empty not allowed)") +
-                    (isNullAllowed ? " (null config allowed)" : " (null not allowed)");
+            String base = validString.validStrings.isEmpty() ? "any non-duplicate values" : validString.toString();
```

Comment:
The following configs don't have a validator, but it seems that they shoud.

BrokerSecurityConfigs:
ssl.cipher.suites
sasl.enabled.mechanisms
sasl.kerberos.principal.to.local.rules
sasl.oauthbearer.expected.audience


DefaultConfigPropertyFilter
config.properties.exclude

DefaultTopicFilter
topics
topics.exclude